desktop_version/src/v6ap.cpp

```diff
@@ -176,7 +176,9 @@ void V6AP_PrintNext() {
 }
 
 void V6AP_AdjustMusic(int* x) {
-    *x = map_music.at(*x);
+    if (auto it = map_music.find(*x); it != map_music.end()) {
```

Comment:
Replace condition with the simpler `map_music.count(*x)`

Reply:
Ah, I found this snippet as supposedly idiomatic for accessing an element if it exists, I can change that.
Should I access with `.at`, or `[]` if the check succeeds?

Reply:
Should use `[]` now, as there is no longer a case where the access fails.

Reply:
I just compared the generated code: https://cpp.godbolt.org/z/x3jx7oj5f
This existing snippet does generate the best code, `[]` access is actually atrocious (inlines RB-tree insertion code), but `.at()` is okay.